_posts/2019-01-23-multifile-python-functions.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Multifile python functions
+description: "Tidy python in an OpenFaaS world; or, what happens when you want to split your Python function into multiple modules?"
```

Comment:
How about a reference to Marie Kondo?

```
Are your Python functions bringing you joy?
```

https://www.bbc.co.uk/news/world-46949722

Reply:
Yes! great suggestion